docs/source/Learn/makingModules/makingModules-2.rst

```diff
@@ -37,6 +37,8 @@ To create message objects in C and C++ modules, you need to run ``python3 conanf
 
 Running ``python3 conanfile.py`` re-creates the IDE project file that will now include access to the new message definition.
 
+.. caution::
```

Comment:
Minor thing, could you please reduce the length of the first line of the commit message here?  It is too long for
GitHub to render properly.  Otherwise documentation built well.

Reply:
Done and rebased as well.